app/src/main/java/com/duckduckgo/app/global/image/GlobalGlideModule.kt

```diff
@@ -42,6 +43,7 @@ class GlobalGlideModule : AppGlideModule() {
         registry: Registry,
     ) {
         val okHttpClientBuilder = OkHttpClient.Builder()
+            .connectTimeout(700, MILLISECONDS) // rather than 10 seconds
```

Comment:
I don't think this is needed. I tried removing it, and couldn't find a bad side effect of doing so. I think that we now show the placeholder first, before trying to load the _real_ favicon renders this unnecessary.

(I'm also a bit worried it brings in unwanted behaviour elsewhere in the app, so keen to kill it if we can)

Reply:
You are correct, this is a leftover from my testing. It should not be a problem, just will timeout at some point but same as it has been so far.